lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -78,19 +78,26 @@ module.exports = function(
     }
 
     Bluebird.try(() => {
-      if (!SupportedFormatsModel.looksLikeWindowsImage(image.path)) {
-        return false;
+      let message = null;
```

Comment:
Can we refactor this to the following expression, so we get rid of `let` (its always nice to be able to make things inmmutable)?

```js
const message = _.attempt(() => {
    if (SupportedFormatsModel.looksLikeWindowsImage(image)) {
        return messages.warning.looksLikeWindowsImage();
    } else if (SupportedFormatsModel.missingPartitionTable(image)) {
        return messages.warning.missingPartitionTable();
    }
});
```